pkg/pool-gyro/coverage.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
```

Comment:
Is this a soft-link? Or is it different and can't be?

Reply:
https://stackoverflow.com/questions/8967902/why-do-you-need-to-put-bin-bash-at-the-beginning-of-a-script-file#:~:text=Adding%20%23!%2Fbin%2Fbash%20as,or%20%22sha%2Dbang%22.

This is just a convention, to tell the OS to run the script in bash.